packages/app/components/Nav/NavBar.tsx

```diff
@@ -119,16 +125,55 @@ export default function NavBar({ courseId }: NavBarProps): ReactElement {
     setVisible(false);
   };
 
+  const courseSelector = (
+    <Menu>
+      {profile?.courses.map((c) => (
+        <Menu.Item key={c.course.id}>
+          <Link href="/course/[cid]/today" as={`/course/${c.course.id}/today`}>
+            {c.course.name}
+          </Link>
+        </Menu.Item>
+      ))}
+    </Menu>
+  );
+
+  const tabs: NavBarTabsItem[] = [
+    {
+      href: "/course/[cid]/today",
+      as: `/course/${courseId}/today`,
+      text: "Today",
+    },
+    {
+      href: "/course/[cid]/schedule",
+      as: `/course/${courseId}/schedule`,
+      text: "Schedule",
+    },
+  ];
+  if (queueId) {
+    tabs.push({
+      href: "/course/[cid]/queue/[qid]",
+      as: `/course/${courseId}/queue/${queueId}`,
+      text: "Queue",
+    });
+  }
+
   return (
     <Nav>
       <LogoContainer>
-        {course && (
-          <Logo href={`/course/${courseId}/today`}>{course.name}</Logo>
-        )}
+        <SimpleDropdown overlay={courseSelector}>
```

Comment:
Since the majority of students will only have one class they are using this office hours app with, should we only render the dropdown if they have more than one course?

Reply:
agree